### ISSUE TITLE
Missing details in video Properties? (Windows)

### ISSUE THREAD

**Issue by nintendraw** [yt-dlp/yt-dlp#9430]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm asking a question and **not** reporting a bug or requesting a feature
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Please make sure the question is worded well enough to be understood

Whenever my yt-dlp output file is the result of a merge (includes the default option) on Windows 10, its Properties -> Details tab seems to be blank or otherwise contains inaccurate information. [This video](https://www.youtube.com/watch?v=okK-wEt7LD0), for example, is 3640x2160px and 60fps at max. `yt-dlp <URL>` and `yt-dlp -f "bv*+ba/b" <URL>` both show this in the resulting file's Properties (only the video length is shown; bitrate seems inaccurate at 0kbps and everything else is blank). I need to add `-k` to get the pre-merge video file in order to tell the resolution and fps. How can I automatically put that sort of information into Properties? Thanks!

![image](https://github.com/yt-dlp/yt-dlp/assets/74480507/e1a294a3-bfea-40de-b3d0-f16acb3f68b1)

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', 'https://www.youtube.com/watch?v=okK-wEt7LD0']
[debug] User config "C:\Users\lisan\AppData\Roaming\yt-dlp\config.txt": ['-o', '~/AppData/Roaming/yt-dlp/Output/%(title)s.%(ext)s']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2024.03.10 from yt-dlp/yt-dlp [615a84447] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 2024-03-11-git-3d1860ec8d-essentials_build-www.gyan.dev (setts), ffprobe 2024-03-11-git-3d1860ec8d-essentials_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.1.0, certifi-2024.02.02, mutagen-1.47.0, requests-2.31.0, sqlite3-3.35.5, urllib3-2.2.1, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets
[debug] Loaded 1803 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: stable@2024.03.10 from yt-dlp/yt-dlp
yt-dlp is up to date (stable@2024.03.10 from yt-dlp/yt-dlp)
[youtube] Extracting URL: https://www.youtube.com/watch?v=okK-wEt7LD0
[youtube] okK-wEt7LD0: Downloading webpage
[youtube] okK-wEt7LD0: Downloading ios player API JSON
[youtube] okK-wEt7LD0: Downloading android player API JSON
[youtube] okK-wEt7LD0: Downloading m3u8 information
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, size, br, asr, vext, aext, hasaud, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] okK-wEt7LD0: Downloading 1 format(s): 315+251
[debug] Invoking http downloader on "https://rr3---sn-nx57ynsl.googlevideo.com/videoplayback?expire=1710266044&ei=XELwZYDcELWlsfIPnIyGgAo&ip=97.94.0.127&id=o-ACGG6Uh0-_-IhoLZzgs46Ybw-HAwNJV1NMV0RmLGaCx3&itag=315&source=youtube&requiressl=yes&xpc=EgVo2aDSNQ%3D%3D&mh=tH&mm=31%2C26&mn=sn-nx57ynsl%2Csn-a5mekn6z&ms=au%2Conr&mv=m&mvi=3&pl=19&gcr=us&initcwndbps=1555000&vprv=1&svpuc=1&mime=video%2Fwebm&gir=yes&clen=80298558&dur=63.113&lmt=1677113451266601&mt=1710244145&fvip=5&keepalive=yes&fexp=24007246&c=IOS&txp=6219224&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cxpc%2Cgcr%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AJfQdSswRgIhAPeiEOC2Wzj4wgmU0QdpRT73GVbnF1w6tIuGoe3WzayZAiEAhfWUgwBo9bieYASooWx4Jr3Wq72QLh54HzpCvD7eGec%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=APTiJQcwRAIgGQy9mpME-ZhZRs1Upl_MWInEI7FCUq-SRp3tOJ_9WngCIBjP6NzEAA2-h05SxPKL1V7NrS9xL2fsrSDsGaMqCUxO"
[debug] File locking is not supported. Proceeding without locking
[download] Destination: C:\Users\lisan\AppData\Roaming\yt-dlp\Output\[The Legend of Zelda Animation⧸MMD] BACK TO YOU.f315.webm
[download] 100% of   76.58MiB in 00:00:02 at 33.05MiB/s
[debug] Invoking http downloader on "https://rr4---sn-nx5s7nel.googlevideo.com/videoplayback?expire=1710266044&ei=XELwZcr1HuyQsfIP1_uhqAg&ip=97.94.0.127&id=o-ALK6SqAPJ3Eh7OA1gh6nsmGdX-xIxK3AXmHsouHYz3G-&itag=251&source=youtube&requiressl=yes&xpc=EgVo2aDSNQ%3D%3D&mh=tH&mm=31%2C29&mn=sn-nx5s7nel%2Csn-nx57ynsl&ms=au%2Crdu&mv=m&mvi=4&pl=19&gcr=us&initcwndbps=1468750&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=1018205&dur=63.141&lmt=1677113395600787&mt=1710244145&fvip=3&keepalive=yes&fexp=24007246&c=ANDROID&txp=6218224&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cxpc%2Cgcr%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AJfQdSswRQIhAM8vRyYgjL9lJWoiDbsqBECh6HMarRvkiFOaQkQIVOXLAiAgmo6WhsAZHM_tlGmfZTlNya0GyYnbmCivgTJxAzYDjg%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=APTiJQcwRAIgBEZBZthpew3ih_pN6Dl0UkXXf64oKCZ7cpQTeHGgsGcCIA1dsTeS02gXaCbIKM9V_jQ0f3zu0H7YgtIzVwVidu-9"
[download] Destination: C:\Users\lisan\AppData\Roaming\yt-dlp\Output\[The Legend of Zelda Animation⧸MMD] BACK TO YOU.f251.webm
[download] 100% of  994.34KiB in 00:00:00 at 5.31MiB/s
[Merger] Merging formats into "C:\Users\lisan\AppData\Roaming\yt-dlp\Output\[The Legend of Zelda Animation⧸MMD] BACK TO YOU.webm"
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -i "file:C:\Users\lisan\AppData\Roaming\yt-dlp\Output\[The Legend of Zelda Animation⧸MMD] BACK TO YOU.f315.webm" -i "file:C:\Users\lisan\AppData\Roaming\yt-dlp\Output\[The Legend of Zelda Animation⧸MMD] BACK TO YOU.f251.webm" -c copy -map "0:v:0" -map "1:a:0" -movflags "+faststart" "file:C:\Users\lisan\AppData\Roaming\yt-dlp\Output\[The Legend of Zelda Animation⧸MMD] BACK TO YOU.temp.webm"
Deleting original file C:\Users\lisan\AppData\Roaming\yt-dlp\Output\[The Legend of Zelda Animation⧸MMD] BACK TO YOU.f251.webm (pass -k to keep)
Deleting original file C:\Users\lisan\AppData\Roaming\yt-dlp\Output\[The Legend of Zelda Animation⧸MMD] BACK TO YOU.f315.webm (pass -k to keep)
```

**Comment by seproDev** [comment]
This is just Windows being bad and not being able to properly parse the files. You can use mediainfo or ffprobe to look up the video metadata.

**Comment by nintendraw** [comment]
"mediainfo" is a bit less easy for me to find... Ahead of actually trying either (yt-dlp's readme is a Lot to read for a non-technical person; I worry about finding the same for mediainfo or ffprobe), would something like `--postprocessor-args Metadata` let me auto-input that stuff into the Details tab?

**Comment by pukkandan** [comment]
No. The problem is not that the data doesn't exist, but that windows is unable to read it from webm. You can:
* download h264 which windows can read
* Use tools like [icaros](https://icaros.en.softonic.com) that enhances windows explorer's metadata processing capabilities
* Use some other tool like ffprobe to lookup metadata

**Comment by seproDev** [comment]
[MediaInfo](https://mediaarea.net/en/MediaInfo) is both a CLI and a GUI program. I use it pretty regularly to inspect files.
![image](https://github.com/yt-dlp/yt-dlp/assets/4618135/d0e4b507-497c-43f1-87da-081421ed3d05)
You just right-click the file and view in MediaInfo. I am not sure how this could get any easier.

**Comment by nintendraw** [comment]
The way I found to get at least YouTube videos to download bv+ba/b videos with metadata included was to add `-S +codec:avc:m4a` to the command line, e.g. `yt-dlp -S +codec:avc:m4a -f bv+ba/b <URL>`. I did lift the addition from the program's readme page, [Sorting Formats](https://github.com/yt-dlp/yt-dlp?tab=readme-ov-file#sorting-formats). I've only just discovered it and so have no idea if it will be applicable to "all" video websites. Thank for the help all!
